### DISCUSSION TITLE
General Q&A

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#8] [Q&A]
Submit general and miscellaneous questions here

**Comment by Squryl** [comment]
🔄️ What does this button do? I assume it's upscale-related being on the top row...bulk/all four? Perhaps at some discount scale-rate or simply just to hit one button instead of pecking for up to four?

**Reply by willwulfken** [reply]
It's a re-roll button. I'm pretty sure it just generates the same prompt again.

**Comment by radical576** [comment]
Is there an easy way to script prompting from a list or a function? I understand that MJ does not support an API at the moment but say I wanted to do a more granular series of something like every --iw from -2 to 2 by 0.1
I realize this is probably more of a discord question but not sure if you have already dealt with this for this application so thought I would ask

**Reply by willwulfken** [reply]
Technically yes, but you'd have to ask them directly if it's okay or against the rules. Because of my uncertainty with that, I just use regex to create prompts.

You could make a script to output a list of prompts

**Reply by willwulfken** [reply]
For example, if you take the following prompts:
```
https://s.mj.run/TTqVVSudxZg Mandelbulb --iw 1

sphere
Californian Gold
```

and apply these regexes:
Find: `^([^\n].*)$`
Replace: `/imagine prompt:$1`

It will output this:
```
/imagine prompt:https://s.mj.run/TTqVVSudxZg Mandelbulb --iw 1

/imagine prompt:sphere
/imagine prompt:Californian Gold
```

You can also add something like a seed:
`/imagine prompt:$1 --seed 4776`

**Reply by hot2hot** [reply]
Hello. I have generated image https://i.postimg.cc/j5BWrkYc/photo-2023-03-01-15-40-10.jpg
I need to make exactly the same, so that only the lion is red. How can I do that?

And then the picture is constantly changing completely. Help me please. I won't be in debt)

**Reply by willwulfken** [reply]
Here's a link to the downloadable style lists: https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/releases/tag/1.0
